Tests/SwiftParserTest/translated/InvalidTests.swift

```diff
@@ -237,35 +298,43 @@ final class InvalidTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(message: "expected ':' and type in parameter", fixIts: ["insert ':' and type"])
-      ]
+      ],
+      fixedSource: """
+        do {
+          class Starfish {}
+          struct Salmon {}
+          func f(s Starfish: <#type#>,
```

Comment:
It would be nice if we were offering to insert a `:` in between `s` and `Starfish`.

Reply:
https://github.com/apple/swift-syntax/issues/1608